src/Tasks/LDAPGroupSyncTask.php

```diff
@@ -142,22 +149,26 @@ public function run($request)
 
         $end = time() - $start;
 
-        $this->log(sprintf(
-            'Done. Created %s records. Updated %s records. Deleted %s records. Duration: %s seconds',
-            $created,
-            $updated,
-            $deleted,
-            round($end ?? 0.0, 0)
-        ));
+        Deprecation::withNoReplacement(function () use ($created, $updated, $deleted, $end) {
+            $this->log(sprintf(
+                'Done. Created %s records. Updated %s records. Deleted %s records. Duration: %s seconds',
+                $created,
+                $updated,
+                $deleted,
+                round($end ?? 0.0, 0)
+            ));
+        });
     }
 
     /**
      * Sends a message, formatted either for the CLI or browser
      *
      * @param string $message
+     * @deprecated 2.3.0 Will be removed without equivalent functionality to replace it
      */
     protected function log($message)
     {
+        Deprecation::notice('2.3.0', 'Will be removed without equivalent functionality to replace it');
```

Comment:
Wrap this in withNoReplacement() rather than all the things calling it

Reply:
> Wrap this in withNoReplacement() rather than all the things calling it

As discussed in person it's more appropriate to wrap the things calling it, so that developers calling it from their own code get an appropriate deprecation warning.